### PR TITLE
update links in README.md

### PR DESCRIPTION
**Pull request by DanCech**
README.md includes links to some files which aren't included in the download packages, this PR switches those out for links to github to enhance discoverability.

We may also want to consider including some of these files in the download packages.

I also updated the slack link to https://slack.grafana.com

### CODE REVIEW

**Review by torkelo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,17 +30,17 @@ The Grafana documentation is available at [grafana.com/docs](https://grafana.com
 
 If you're interested in contributing to the Grafana project:
 
-- Start by reading the [Contributing guide](/CONTRIBUTING.md).
-- Learn how to set up your local environment, in our [Developer guide](/contribute/developer-guide.md).
+- Start by reading the [Contributing guide](https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md).
```
Comment (RichiH) [inline README.md:33]:
```suggestion
- Start by reading the [Contributing guide](https://github.com/grafana/grafana/blob/HEAD/CONTRIBUTING.md).
```

Else, we break our links when we move master to main

File: README.md [hunk 2]
```diff
@@ -30,17 +30,17 @@ The Grafana documentation is available at [grafana.com/docs](https://grafana.com
...
+- Learn how to set up your local environment, in our [Developer guide](https://github.com/grafana/grafana/blob/master/contribute/developer-guide.md).
```
Comment (RichiH) [inline README.md:34]:
```suggestion
- Learn how to set up your local environment, in our [Developer guide](https://github.com/grafana/grafana/blob/HEAD/contribute/developer-guide.md).
```

File: README.md [hunk 3]
```diff
@@ -30,17 +30,17 @@ The Grafana documentation is available at [grafana.com/docs](https://grafana.com
...
 - Follow [@grafana on Twitter](https://twitter.com/grafana/).
 - Read and subscribe to the [Grafana blog](https://grafana.com/blog/).
 - If you have a specific question, check out our [discussion forums](https://community.grafana.com/).
-- For general discussions, join us on the [official Slack](http://slack.raintank.io/) team.
+- For general discussions, join us on the [official Slack](https://slack.grafana.com) team.
 
 ## License
 
-Grafana is distributed under [AGPL-3.0-only](LICENSE). For Apache-2.0 exceptions, see [LICENSING.md](LICENSING.md).
+Grafana is distributed under [AGPL-3.0-only](LICENSE). For Apache-2.0 exceptions, see [LICENSING.md](https://github.com/grafana/grafana/blob/master/LICENSING.md).
```
Comment (RichiH) [inline README.md:46]:
```suggestion
Grafana is distributed under [AGPL-3.0-only](LICENSE). For Apache-2.0 exceptions, see [LICENSING.md](https://github.com/grafana/grafana/blob/HEAD/LICENSING.md).
```

**Review by RichiH — DISMISSED** [review]

**Comment by torkelo** [comment]
synced this with main, thing we can merge this

**Pull request merged**